.clang-format

```diff
@@ -6,13 +6,14 @@ AlignConsecutiveAssignments: true
 AlignConsecutiveDeclarations: true
 # AlignEscapedNewlinesLeft: false
 AlignOperands:   true
+AllowShortBlocksOnASingleLine: Empty
```

Comment:
```
> clang-format --version
clang-format version 7.0.1 (tags/RELEASE_701/final 349238)
7012.sektor@sullust:/home/sektor/repos/srt-devel2.git/srtcore
> clang-format utilities.h >NULL
YAML:13:32: error: invalid boolean
AllowShortBlocksOnASingleLine: Empty
                               ^~~~~
Error reading /home/sektor/repos/srt-devel2.git/.clang-format: Invalid argument
```

What is strange is that I can see this option in the master branch currently and its value is `false`.

Reply:
Deleted changes of this option.